src/lit-element.ts

```diff
@@ -32,14 +31,9 @@ export abstract class LitElement extends UpdatingElement {
 
   /**
    * Updates the element. This method reflects property values to attributes
-   * and calls `render` to render DOM via lit-html. It should be overridden to
-   * perform tasks on rendered DOM. Within `update()` setting properties does
-   * not trigger `invalidate()`, allowing property values to be computed and
-   * validated before DOM is rendered and updated. This means in an override
-   * of `update()`, before calling `super.update()` setting properties will not
-   * trigger another update, but after calling `super.update()` setting
-   * properties will trigger another update.
-   * * @param changedProperties Map of changed properties with old values
+   * and calls `render` to render DOM via lit-html. Setting properties inside
```

Comment:
I think these docs should more closely match UpdatingElement's version, ie. mention reflection.

Reply:
Already mentioned, but made more similar.